src/Workspaces/Remote/ServiceHub/Services/CodeAnalysisService_Diagnostics.cs

```diff
@@ -56,11 +56,11 @@ public Task CalculateDiagnosticsAsync(DiagnosticArguments arguments, string stre
 
         public void ReportAnalyzerPerformance(List<AnalyzerPerformanceInfo> snapshot, int unitCount, CancellationToken cancellationToken)
         {
-            RunService(token =>
+            RunService(() =>
```

Comment:
Wait, so @jaredpar was advertising the new language feature where we could have a lambda use the same name as a parameter, and this (which I think was the biggest case?) doesn't need it anymore? :smile:

Reply:
it was needed before since there was an issue in StreamJsonRpc. but they now support it through new CancelLocallyInvokedMethodsWhenConnectionIsClosed feature, so I don't need to do that myself through mergedcancellation token.

unfortunately, the token and cancellation token were actually 2 different cancellations and cause issues if used wrongly. now, no confusion since there is only 1 cancellation token.

Reply:
> unfortunately, the token and cancellation token were actually 2 different cancellations and cause issues if used wrongly. now, no confusion since there is only 1 cancellation token.

Nice!